### DISCUSSION TITLE
botinfo

### DISCUSSION THREAD

**Discussion by miatoszs** [eritislami/evobot#375] [Ideas]
I write a botimfo command you can use it

const Discord = require("discord.js")

const { version } = require("discord.js");
const moment = require("moment");
const m = require("moment-duration-format");
let os = require('os')
let cpuStat = require("cpu-stat")
const ms = require("ms")

module.exports = {
    name: "botinfo",
    
  description: "Bot info",
  async execute(message, args) {  //command
  let cpuLol;
  cpuStat.usagePercent(function(err, percent, seconds) {
      if (err) {
          return console.log(err);
      }
    
      const botinfo = new Discord.MessageEmbed()
          .setAuthor(message.client.user.username)
          .setTitle("__**Status:**__")
          .setColor("RANDOM")
          .addField("⏳ Ram usage", `${(process.memoryUsage().heapUsed / 1024 / 1024).toFixed(2)} / ${(os.totalmem() / 1024 / 1024).toFixed(2)} MB`, true)
          
          .addField("📁 Users", `${message.client.users.cache.size}`, true)
          .addField("📁 Servers", `${message.client.guilds.cache.size}`, true)
          .addField("📁 Channels ", `${message.client.channels.cache.size}`, true)
          .addField("👾 Discord.js", `v${version}`, true)
          .addField("🤖 Node", `${process.version}`, true)
          .addField("🤖 CPU", `\`\`\`md\n${os.cpus().map(i => `${i.model}`)[0]}\`\`\``)
          .addField("🤖 CPU usage", `\`${percent.toFixed(2)}%\``, true)
          .addField("🤖 Architecture", `\`${os.arch()}\``, true)
          .addField("💻 Platform", `\`\`${os.platform()}\`\``, true)
          .addField("API Latency", `${(message.client.ws.ping)}ms`)  
      message.channel.send(botinfo)
  });
  }
  };

**Comment by chiquidev** [comment]
Thanks! Very useful, but I think that this isn't the correctly place to post it...
